pandas/core/tools/datetimes.py

```diff
@@ -604,6 +604,7 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     elif isinstance(arg, ABCIndexClass):
         cache_array = _maybe_cache(arg, format, cache, convert_listlike)
         if not cache_array.empty:
+            errors  = 'ignore' if not utc else errors
```

Comment:
This seems like a bandaid fix. I suspect the fix needs to happen at the cython level.

Reply:
we've reworked the fix, could you please re-check?